c4a-test/build.gradle

```diff
@@ -116,8 +111,7 @@ subprojects {
     }
 
     dockerAlfresco {
-//        baseImage = "docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:${alfresco_img_version}"
-        baseImage = "hub.xenit.eu/alfresco-enterprise/alfresco-repository-enterprise:${alfresco_img_version}"
+        baseImage = "private.docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:${alfresco_img_version}"
```

Comment:
Does this work? Apix uses `baseImage = 'docker.xenit.eu/private/alfresco-enterprise/alfresco-repository-enterprise:7.3.0.1'`

Reply:
yeah, that's new. cloudsmith implemented that on our request. Works around an issue where docker can only have one login per repo.
Now all the old namespaces count as separate repos.
Also avoids an issue where a login to an authenticated repo blocks access to an open repo. (See Thijs for more info)

Reply:
though the private element in the querypath does needs to be dropped

Reply:
Aha! Good to know. They should @channel this kind of stuff :/